wiki/cli.py

```diff
@@ -26,14 +26,19 @@ def main(ctx, directory):
 
 @main.command()
 @click.option('--debug/--no-debug', envvar='WIKI_DEBUG', default=False)
+@click.option('--host', envvar='HOST', default=None)
+@click.option('--port', envvar='PORT', default=None, type=int)
 @click.pass_context
-def web(ctx, debug):
+def web(ctx, debug, host, port):
     """
         Run the web app.
 
         \b
         :param bool debug: whether or not to run the web app in debug
             mode.
+        :param str  host: Set the host to 0.0.0.0 to connect from outside 
+            defaults to 127.0.0.1.
+        :param int  port: Set the listening port defaults to 5000.
```

Comment:
We should use punctuation to better outline the default (e.g. separate sentence).